app/helpers/user_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module UserHelper
+  def tag_name_options      
+    Gutentag::Tag.pluck(:name)
```

Comment:
Eventually, this list could get too long so that we'd want to be choosy about which to include here, but for now listing all seemed fine.